package-configuration/linux-cli/get-cloudify.py

```diff
@@ -122,7 +125,9 @@ def run(cmd, sudo=False):
     # while the process is still running, print output
     while proc.poll() is None:
         if VERBOSE:
-            _print_proc_out(proc)
+            while _print_proc_out(proc):
+                pass
+        time.sleep(1)
     _print_proc_out(proc)
```

Comment:
The second print_proc_out is needed, in the end-case of lines bring written to stdout after the last _print_proc_out (inside the loop) and before proc.poll() returns false. It's quite an end-case, but it's theoretically possible. Therefore, the second call (outside the loop) may return true.

Regarding moving the "while" loop into print_proc_out — I thought about it and even implemented it that way originally, but then decided not to change the API of print_proc_out as I wasn't (and still am not) sure if someone's going to be reusing it for other purposes. I agree though that it'd be a better approach overall.